docs/src/assets/scss/components/alert.scss

```
@@ -58,38 +40,28 @@
 
 .alert__type {
     display: block;
-    font-weight: 500;
+    font-family: "Inter Variable", sans-serif;
+    font-weight: 600;
     margin-bottom: .25rem;
     margin-block-end: .25rem;
 
     .alert--warning & {
-        color: var(--color-rose-700);
-
-        [data-theme="dark"] & {
-            color: var(--color-rose-200);
-        }
+        color: var(--alert-warning-heading-color);
     }
 
     .alert--important & {
-        color: var(--color-warning-700);
-
-        [data-theme="dark"] & {
-            color: var(--color-warning-200);
-        }
+        color: var(--alert-important-heading-color);
     }
 
     .alert--tip & {
-        color: var(--color-success-700);
-
-        [data-theme="dark"] & {
-            color: var(--color-success-200);
-        }
+        color: var(--alert-tip-heading-color);
     }
 }
 
 .alert__learn-more {
     display: block;
-    font-weight: 500;
+    font-family: "Inter Variable", sans-serif;
```

```suggestion
    font-family: "Inter Variable", sans-serif;
```
Why is the `font-family` explicitly specified as `"Inter Variable", sans-serif`? Is this necessary?

---

Actually i tried different fonts from repo and this combination of `font-family: "Inter Variable", sans-serif;` and `font-weight: 600;` giving the same result (font and its weight) we have right now, its also about where we are accessing the font through `@font-face`, if we access the `Inter Variable` font through `base.html` file as we are doing right now, then it's not working.

---

You are probably fixing it in the wrong place? https://github.com/eslint/eslint/blob/db0b844a66ee25483f9619d04346de1a2a0d79fa/docs/src/_includes/layouts/base.html#L135-L148 

this is where the script adds inter semibold to the document. May be we might want to look at what this is doing and then move it to scss approach rather than changing font-family of already existing elements.

---

Lets use scss to load this font as we do for other fonts. If we face any issues we can feel free to make changes its fine.

---

Done!

---

thanks!